Server/Terrain.py

```diff
@@ -0,0 +1,91 @@
+import random as ra
+import math as mat
+import bresenham as br
+
+
+SCREEN_WIDTH = 800
+SCREEN_HEIGHT = 600
+
+BASE_LEVEL = 120
+MIN_LEVEL = 50
+
+MAX_ABS_ANGLE = 15
+NO_ANGLE_BASE_P = 0.45
+
+
+class Terrain:
+    def __init__(self):
+        self.__screen_width = SCREEN_WIDTH
+        self.__screen_height = SCREEN_HEIGHT
+
+        self.__base_level = BASE_LEVEL
+        self.__min_level = MIN_LEVEL
+
+        self.__max_abs_slope = MIN_LEVEL
+        self.__no_slope_base_p = NO_ANGLE_BASE_P
```

Comment:
No idea what "no_slope_base_p" is. Also angle-slope consistency is desirable.

Reply:
As it can be seen in the function "__angle_rand_gen_and_append":
- NO_ANGLE_BASE_P is the base probability of generating an angle equal to 0 (horizontal line);
- it is just a base probability, as 0 can come up even randomly when generating an angle in the interval (-MAX_ABS_ANGLE, MAX_ABS_ANGLE);
- it was introduce because I found that is desirable to have mostly a horizontal terrain.

Reply:
Wouldn't it be better to name it ZERO_ANGLE_BASE_PROB? Or could you place a comment after its declaration?